components/mosart/src/wrm/WRM_subw_IO_mod.F90

```diff
@@ -374,7 +374,7 @@ subroutine WRM_init
            enddo
         enddo
         if (cnt /= cntw) then
-           write(iulog,"(a,3i8)"), subname//'ERROR: sMat g2d cnt errora',cntw,cnt
+           write(iulog,"(a,3i8)") subname//'ERROR: sMat g2d cnt errora',cntw,cnt
```

Comment:
Just out of curiosity. was there a reason not using trim(subname) in line 377, 399, 425, 451, while you used trim(subname) in line 813,847?

Reply:
The inconsistent use of trim is from the original code.